Jenkinsfile

```diff
@@ -65,11 +65,9 @@ pipeline {
                 timeout(time: 3, unit: 'HOURS')
             }
             steps {
-                // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
-                // -DskipITs is temporary to skip all the tests that were failing at the time. See https://github.com/codice/ddf/issues/5777
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
                     sh 'mvn install -B -DskipStatic=true -DskipTests=true $DISABLE_DOWNLOAD_PROGRESS_OPTS'
-                    sh 'mvn clean install -B -DskipITs -pl !$ITESTS -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                    sh 'mvn clean install -B -Pcx-internal-ci -pl !$ITESTS -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
@TonyMorrison can you help connect the dots for me, on what the `cx-internal-ci` profile accomplishes? That one is new to me 😄

Reply:
it is a profile in the global-maven-settings in which i put a property, skipITs set to true. In the pom i added a new configuration for skipITs with the value coming from this property. This makes skipITs true by default if you use the cx-internal-ci. 
For what I am doing, this makes (theoretical, unless I am accidentally getting the results via something else) skipITs true for stages where I wanted it and in the one stage where we wanted the ITests we just set the property to false.

Reply:
Would it make more sense to call the profile `skipITs`? Is there anything else in the profile?